conanfile.py

```diff
@@ -108,26 +108,14 @@ class BasiliskConan(ConanFile):
             except:
```

Comment:
Dropping an approval on this PR with the following notes:

- [ ] usingProtobuf.cmake is repeated in the first commit message and I don't think the back ticks are needed as they will not be rendered anywhere.

- [ ] The first and second commits need to be squashed together. The first commit doesn't build/isn't complete without the changes in the second commit. Once squashed they make up a truly single atomic commit.